### PR TITLE
Mentioned that Float cannot represent negative zero

### LINKED ISSUE #24088: DOC: Mention that Float cannot represent negative zero
The documentation for sympy.Float mentions that the Float type cannot represent infinity and NaN. Given that IEEE 754 floats also have separate values for positive and negative zero, but sympy Float does not (not even if passing mpf tuple `(1,0,0,0)` to the constructor), the documentation should mention this difference.

### PR DESCRIPTION
**Pull request by metabiswadeep**
#### Release Notes

- other
  - an addition to documentation about Float.

### CODE REVIEW

**Comment by oscargus** [comment]
Please move this to the Note section.

**Comment by oscargus** [comment]
I suggest that you put it last. Now it is in the middle of the approximation discussion.

File: sympy/core/numbers.py [hunk 1]
```diff
@@ -1015,6 +1015,8 @@ class Float(Number):
     -oo
     >>> _.is_Float
     False
+
+    Zero in Float only has a single value. Values are not seperate for positive and neghative zeroes.
```
Comment (chatcannon) [inline sympy/core/numbers.py:1019]:
```suggestion
    Zero in Float only has a single value. Values are not separate for positive and negative zeroes.
```

**Review by chatcannon — CHANGES_REQUESTED** [review]
I spotted some typos but otherwise it is what I was looking for.

**Pull request merged**